### ISSUE TITLE
FSR_comparision

### ISSUE THREAD

**Issue by youngAt19** [yangxy/GPEN#54]
Hi, thanks for sharing your nice work.
Since pSp trained their method at resolution_256 (output of generator is at resolution_1024, then downsampled to 256 for calculating losses), 
I tried to reimplement the pSp super-resolution at resolution_1024 but the result is not as good as that at resolution_256.
Did you reimplement the FSR models (likes pSp), or just use the models trained by the original authors?
Could you please share some training details if reimplement? 
Or can I directly use their trained model to get the output of generator at resolution_1024 for comparision?

**Comment by yangxy** [comment]
1. I just used their pre-trained model.
2. I want to. But we are selling this technology at present. I am not allowed to do so.
3. I think so.

The pSp tries to embed a real image into a latent code of StyleGAN. This Inevitably leads to loss of the details of the original image. So if the input is high-quality, the output of pSp may dislike the input. I guess that is why they need to resize the input image to 256x256 (in order to degrade any possible high-quality inputs). 
Thanks to the U-shaped structure and the finetuning process, GPEN does not suffer from this issue. Please check the seg2face example, the output of GPEN is completely consistent with the input, while the output of pSp is not.

**Comment by youngAt19** [comment]
Get it! Thanks a lot.

**Comment by sd707589** [comment]
@youngAt19 @yangxy Pardon, what is the 'pSp' or the 'FSR'?  Can you share the github link?
Thx
